sky/adaptors/kubernetes.py

```diff
@@ -18,6 +18,8 @@
 urllib3 = common.LazyImport('urllib3',
                             import_error_message=_IMPORT_ERROR_MESSAGE)
 
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
This will cause urllib3 being imported when the adaptors being imported, which may cause errors when skypilot is not installed with `skypilot[kubernetes]` or `skypilot[all]`.

Reply:
Actually, we only specify `urllib3` in the dependency of `aws`, we should have it specified for `kubernetes` as well in `setup.py`

Reply:
Good point, I have moved it to `_load_config` so that it is invoked only  when a) `_configured=False` and b) other import checks have already been run.

urllib3 does not need to be explicitly specified in setup.py, since the `kubernetes` package depends on urllib3 and urllib3 will automatically be installed when `kubernetes` is installed.